golem_core/mid/simple_scorer.py

```diff
@@ -81,8 +87,11 @@ async def _wait_until_ready(self) -> None:
         start = datetime.now()
 
         while True:
+            await asyncio.sleep(0.1)
+            if self._min_wait is not None and  datetime.now() - start < self._min_wait:
+                # force wait until time exceeds `min_wait`
+                continue
             if self._min_proposals is None or len(self._scored_proposals) >= self._min_proposals:
                 break
             if self._max_wait is not None and datetime.now() - start >= self._max_wait:
```

Comment:
As `datetime.now()` is called twice, introduce variable to cache its value, or even result of `datetime.now() - start`.

Reply:
Added